### PR TITLE
Add, edit, list sequence diagram

### PR DESCRIPTION
**Pull request by wish2023**

### CODE REVIEW

**Review by daniellimws — APPROVED** [review]
The sequence diagram doesn't need to end with a cross at the end of the lifeline? But this is nice!

File: docs/sequenceDiagrams/AddSequenceDiagramCode.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title Add
+
+AddCommand->+TaskList: addTask()
+
```
Comment (Kafcis) [inline docs/sequenceDiagrams/AddSequenceDiagramCode.txt:5]: need an activation bar here

activate AddCommand

File: docs/sequenceDiagrams/EditSequenceDiagramCode.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+TaskList->TaskList: checkClash(possibleEditedTask)
+activate TaskList
+deactivate TaskList
+
+
+TaskList->+Task: edit()
+Task-->-TaskList:
+TaskList-->-EditCommand:
+deactivate TaskList
```
Comment (Kafcis) [inline docs/sequenceDiagrams/EditSequenceDiagramCode.txt]: Cleaned up the diagram a bit

```suggestion
title Edit



EditCommand->+TaskList: editTask()
activate EditCommand

TaskList->TaskList: getTaskFromNumber()
activate TaskList
deactivate TaskList

TaskList->-TaskList: createTask()
activate TaskList
TaskList->*Task:
activate Task
deactivate Task


TaskList->TaskList: checkClash(possibleEditedTask)
activate TaskList
deactivate TaskList


TaskList->+Task: edit()
Task-->-TaskList:
TaskList-->-EditCommand:
deactivate TaskList
```

File: docs/sequenceDiagrams/ListSequenceDiagramCode.txt [hunk 3]
```diff
@@ -0,0 +1,31 @@
+title List
```
Comment (Kafcis) [inline docs/sequenceDiagrams/ListSequenceDiagramCode.txt:1]: Nicely done. one small edit will be making the activation bar for the ListCommand
avtivate ListCommand

**Review by Kafcis — CHANGES_REQUESTED** [review]

**Pull request merged**